dts/bindings/rng/st,stm32-rng.yaml

```diff
@@ -16,3 +16,15 @@ properties:
 
     clocks:
       required: true
+
+    htcfg1:
```

Comment:
```suggestion
    health-test-magic:
```

Reply:
I changed the description, but I would prefer to keep the name `htcfg` as it is the bitfield name used in Reference Manual.
Similarly I would prefer to keep  `htcfg1` as it refer to htcfg from Reference Manual and it is also used in STM32 HAL.
ex:
```
#define RNG_HTCFG_1   0x17590ABCU
#define RNG_HTCFG     0x000CAA74U
```

Reply:
I saw that htcfg and htcfg1 are the register section names coming from the Ref Manual, 
I asked this change on purpose because: 
- When possible, using human readable names makes life easier to everyone. It could happen that sometime, register name is transparent and helps giving an idea of its content, but it's not the case here. You can see that in general, as a convention, device tree property names are trying to be self explanatory.
- "health test configuration" could also be found in the reference manual
- There is no guarantee that this name will persist across series and it might happen that for an upcoming series, a slightly different name is used. And then we would have a clear and unfortunate disconnection between device tree property and reference manual and all benefit lost.

Reply:
OK. Done.